test/fivetran-destination/test-writes/02-verify.td

```diff
@@ -10,9 +10,6 @@
 > SELECT k1, k2, v1, v2, _fivetran_deleted FROM test.tester.test_writes
 k1  k2  v1     v2               _fivetran_deleted
 -------------------------------------------------
-1   a   12.78  "{\"x\":\"y\"}"  false
-1   b   91.28  {}               false
-2   a   34.21  null             false
-
-> SELECT COUNT(DISTINCT(_fivetran_synced)) FROM test.tester.test_writes
```

Comment:
The way the updated tester handles the `_fivetran_synced` column we can't really make any assertions about it, so I removed all of these checks